akka-stream/src/main/scala/akka/stream/ActorMaterializer.scala

```diff
@@ -287,10 +287,11 @@ object ActorMaterializerSettings {
     outputBurstLimit:            Int,
     fuzzingMode:                 Boolean,
     autoFusing:                  Boolean,
-    maxFixedBufferSize:          Int) =
+    maxFixedBufferSize:          Int,
+    alwaysLogErrors:             Boolean) =
```

Comment:
The way to work around the compat issue here is to add a second create method with the existing signature